library/src/lib/time/time.component.ts

```diff
@@ -22,7 +23,7 @@ import { TimeI18nLabels } from './i18n/time-i18n-labels';
 })
 export class TimeComponent implements OnChanges, ControlValueAccessor {
 
-    /** @Input When set to true, uses the 24 hour clock (hours ranging from 0 to 23)
+    /** @Input When set to false, uses the 24 hour clock (hours ranging from 0 to 23)
      * and does not display a period control. */
     @Input() meridian: boolean = false;
```

Comment:
gotta document as breaking

Reply:
The comment was just not correct. Meridian option works like before.